source/macos/AdaptiveCards/AdaptiveCards/Renderers/FactSetRenderer.swift

```diff
@@ -29,45 +29,51 @@ class FactSetRenderer: NSObject, BaseCardElementRendererProtocol {
         var requiredWidth: CGFloat = 0
         // Main loop to iterate over Array of facts
         for fact in factArray {
-            let markdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig)
-            let markdownString = TextUtils.getMarkdownString(for: rootView, with: markdownParserResult)
-            let attributedContent = TextUtils.addFontProperties(attributedString: markdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
+            let valueViewMarkdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig, isTitleField: false)
+            let valueViewMarkdownString = TextUtils.getMarkdownString(for: rootView, with: valueViewMarkdownParserResult)
+            let valueViewAttributedContent = TextUtils.addFontProperties(attributedString: valueViewMarkdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
```

Comment:
```suggestion
            let valueMarkdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig, isTitleField: false)
            let valueMarkdownString = TextUtils.getMarkdownString(for: rootView, with: valueViewMarkdownParserResult)
            let valueAttributedContent = TextUtils.addFontProperties(attributedString: valueViewMarkdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
```

`View` can be omitted in these variable names.